concordium-sdk/src/main/java/com/concordium/sdk/ClientV2MapperExtensions.java

```diff
@@ -369,7 +368,7 @@ static PendingChange to(StakePendingChange pendingChange) {
                         .build();
             default:
             case CHANGE_NOT_SET:
```

Comment:
In what cases is this case reached? Who has to fail?

Reply:
As per my understanding, then it is reached if the pending change is not set. 
https://protobuf.dev/reference/java/java-generated/#oneof-fields

But before calling this function we always make sure that it is indeed set.

However, cleaned this up now, so this function returns `Optional<..>` and the `Optional.empty()` is returned via the case mentioned.